config/vufind/searches.ini

```diff
@@ -861,3 +865,12 @@ maxLimit = 100
 ; ttl, Time to Live, i.e. cache entry life time in seconds. 300 seconds by default.
 ;options[ttl] = 300
 
+; This section provides hidden filters for AbstractSearchObject recommendation
+; module whose backend search behavior is configured in this file.
+[AbstractSearchObjectHiddenFilters]
+
+; Hidden filters are defined as in [RawHiddenFilters], i.e. each is an array in
+; order to handle more complex filters.  The key must match a [hidden filters key]
+; in a recommendation module, defined in the configuration file for the primary
+; search results.
```

Comment:
I cannot manage to write this in a way that makes sense.  I know it's Friday afternoon but still.  If you understand it maybe you can suggest some better wording.  If not I will explain.

Reply:
Would this be less confusing if you specified an .ini section name in the recommendation module configuration, instead of having sections inside a section? That is, what if rather than `video[]` inside `[AbstractSearchObjectHiddenFilters]`, you just did `[CatalogResultsVideoFilter]`? Then you could just document it as an example, and reference the example from the module documentation, but there's nothing hard-coded or deeply nested.